README.md

```diff
@@ -28,28 +28,35 @@ for Rackspace Private Clouds.
 
 Basic Setup:
 
-1. Complete an installation of
-[os-ansible-deployment](https://github.com/stackforge/os-ansible-deployment)
-as per the standard mechanism described there.
-2. Clone this repository.
-3. Recursively copy `rpc-extras/etc/openstack_deploy/*` to
+1. Clone 
+[os-ansible-deployment](https://github.com/stackforge/os-ansible-deployment).
+2. Clone [rpc-extras](https://github.com/rcbops/rpc-extras).
+3. From the root of the os-ansible-deployment clone, execute
```

Comment:
Just as a small point, it seems like 4 could well be done before 3. That is also the order they are executed by gate-check-commit.sh and unless we have a reason to deviate I'm not sure why we should.